volatility3/framework/symbols/linux/__init__.py

```diff
@@ -830,3 +834,97 @@ def get_cached_pages(self) -> interfaces.objects.ObjectInterface:
             page = self.vmlinux.object("page", offset=page_addr, absolute=True)
             if page:
                 yield page
+
+
+@dataclasses.dataclass
+class TimespecVol3(object):
```

Comment:
Again, not sure what means it needs Vol3 in the title of the class?  Seems oddly specific and out of character for any other classes have no.  Not an overly descriptive class title if someone just saw it somewhere.  How/why is this class different to a standard datetime object, or a `timespec64` or any of a myriad other time holding data structures?

Reply:
Why aren't all these helper methods of a `timespec64` class override?

Reply:
I've been trying to create a dummy object within the framework but ran into issues. I attempted to replicate what's done in [object()](https://github.com/volatilityfoundation/volatility3/blob/develop/volatility3/framework/contexts/__init__.py#L125) using clone, ObjectInformation, etc., but no luck. 
Could you explain how to create a timespace64 object that allows updating values without being attached to an offset/address? If that's possible, I could relocate this code within the timespace64 class; otherwise, it'll need to stay as a separate class.

Reply:
That's ok.  It feels like you could create a class override that implents a `timespec64` interface, which just contains the necessary data as attributes.  For volatility created objects, those would be derived from the layer/offset and data read from the layer.  For others it could just be the storage of the two integers.  So the interface would contain the code that operated just one the two values, the class override would inherit from the interface, and then a stub class that instantiates the interface could be used to make arbitrary ones?  So:

```
Timespec64Interface - all the methods you actually want to work with
Timespec64(Timespec64Interface, Structure) - Class override applied to Timespec64 ISF data
Timespec64Concrete(Timespec64Interface) - Concrete implementation instantiated by the two variables, rather than a structure)
```

The name would need some thinking about, perhaps it should be Timespec64Abstract because it's just the concept of a timespec64, but the idea's the same.  Something you can use to make local timespec64-ish things.  Since both would adhere to the interface, the interface is where all the methods would go.

Reply:
Brilliant, did the trick!. It looks a lot cleaner and simpler now. Hope it's just what you wanted! 
This also includes `__sub__()` implemented as `__add__()` + `negate()` for your enjoyment  ;)

Reply:
Yep, looks good, thanks for clearing that up!  Happy if you want to change the names, but at least they should be descriptive...